### PR TITLE
[docs] fix: fix docs for ascend

### PR DESCRIPTION
**Pull request by phdddd**
### What does this PR do?

Special modifications were made to the Ascend documentation, including modifications to the Ascend homepage and some related materials.

### Checklist Before Starting

- Search for relative PRs/issues and link here: ...
- PR title follows `[{modules}] {type}: {description}` format (enforced by [check_pr_title.yml](.github/workflows/check_pr_title.yml))
  - **Allowed modules:** `agent`, `ci`, `ckpt`, `config`, `data`, `dist`, `docker`, `docs`, `logging`, `misc`, `model`, `omni`, `optim`, `ops`, `parallel`, `perf`, `release`, `task`, `trainer`
  - **Allowed types:** `feat`, `fix`, `refactor`, `chore`, `test`
  - Breaking changes: prepend `[BREAKING]` — e.g. `[BREAKING][parallel, model] feat: dynamic batching`

### Test

> Validation results (training curves, eval metrics) for changes not covered by CI.

### API and Usage Example

> Show API changes and usage examples if applicable.

### Design & Code Changes

> High-level design description and specific change list.

### Checklist Before Submitting

- Read the [Contribute Guide](https://github.com/ByteDance-Seed/VeOmni/blob/main/CONTRIBUTING.md)
- Applied pre-commit checks
- Added/updated documentation
- If `tasks/` training scripts were moved or renamed: updated `docs/` examples and verified `python3 scripts/ci/check_doc_task_paths.py` passes (also enforced by the **Check doc task paths** CI workflow)
- Added tests to CI workflow (or explained why not feasible)

### CODE REVIEW

File: docs/hardware_support/AscendDockerUsage/build_a2_docker.md
Comment (Crystal-jiang) [inline docs/hardware_support/AscendDockerUsage/build_a2_docker.md:1]: It might be worth considering moving it to the directory under hardware_support.

File: docs/index.md [hunk 1]
```diff
@@ -12,12 +12,11 @@ VeOmni is a versatile framework for both single- and multi-modal pre-training an
...
 ```{toctree}
-:maxdepth: 1
+:maxdepth: 2
 :caption: Get Started
 
-get_started/installation/install.md
-get_started/installation/install_ascend_x86.md
-get_started/installation/install_ascend_arm.md
+Installation <get_started/installation/index>
+Ascend Docker Use <get_started/AscendDockerUse/index>
```
Comment (Crystal-jiang) [inline docs/index.md]: Ascend Docker Usage

File: docs/index.md [hunk 2]
```diff
@@ -12,12 +12,11 @@ VeOmni is a versatile framework for both single- and multi-modal pre-training an
...
 
 ```{toctree}
-:maxdepth: 1
+:maxdepth: 2
 :caption: Get Started
 
-get_started/installation/install.md
-get_started/installation/install_ascend_x86.md
-get_started/installation/install_ascend_arm.md
+Installation <get_started/installation/index>
```
Comment (Crystal-jiang) [inline docs/index.md]: This syntax is not supported in ReadTheDocs.

File: docs/get_started/installation/index.md
Comment (Crystal-jiang) [inline docs/get_started/installation/index.md:1]: This file seems redundant; you can just write it directly into the main directory.

File: docs/hardware_support/get_started_npu.md [hunk 3]
```diff
@@ -1,22 +1,68 @@
...
-| [Wan2.1](../examples/wan2.1.md)     | 14B        | ✅       | ✅     |       |    | ✅  |      |
+VeOmni supports a wide range of models on Ascend NPUs, including large language models, multimodal models, and diffusion models. Below is a comprehensive list of supported models with their features:
+
+| Model                               | Model Size                    | Support | FSDP1 | FSDP2 | EP | SP | Note |
+|-------------------------------------|-------------------------------|---------|-------|-------|----|----|------|
+| [DeepSeek2.5/3/R1](https://huggingface.co/deepseek-ai) | 236B/671B                     | ✅       | ✅     | ✅     |    |    |      |
+| [Llama3-3.3](https://huggingface.co/meta-llama) | 1B/3B/8B/70B                  | ✅       | ✅     | ✅     |    |    |      |
+| [Qwen2-3](https://huggingface.co/Qwen) | 0.5B/1.5B/3B/7B/14B/32B/72B   | ✅       |       | ✅     |    |    |      |
+| [Qwen3](../examples/qwen3.md)       | 8B                            | ✅       |       | ✅     |    |    |      |
+|                                     | 30B                           | ✅       |       | ✅     |    |    |      |
```
Comment (Crystal-jiang) [inline docs/hardware_support/get_started_npu.md]: Merge 8B and 30B into a single line.

File: docs/hardware_support/AscendDockerUsage/build_a2_docker.md [hunk 4]
```diff
@@ -0,0 +1,263 @@
...
+## Step 2: Build the Custom Image
+Build the VeOmni Ascend A2 image using the appropriate Dockerfile for your architecture.
+
+**Note:** Proxy settings are optional and only needed if your server requires proxy access to the internet. Remove the proxy arguments if not needed.
+
+### For x86 Architecture
+
+```bash
+docker build \
+  # Optional proxy settings (remove if not needed)
```
Comment (FoolPlayer) [inline docs/hardware_support/AscendDockerUsage/build_a2_docker.md]: Comment should not be followed by bash command, all the code after that will be ignored.

File: docs/hardware_support/AscendDockerUsage/build_a2_docker.md [hunk 5]
```diff
@@ -0,0 +1,263 @@
...
+  --build-arg http_proxy=http://<user>:<pass>@<host>:<port> \
+  --build-arg https_proxy=http://<user>:<pass>@<host>:<port> \
+  --build-arg no_proxy=localhost,127.0.0.1 \
+  -t ascend-a2-env:v1 \
+  -f docker/ascend/Dockerfile.ascend_8.3.rc2_a2 \
+  .
```
[the remark this answers is not in the document]
Comment (FoolPlayer) [reply docs/hardware_support/AscendDockerUsage/build_a2_docker.md:44]: Agree!

File: docs/hardware_support/get_started_npu.md [hunk 6]
```diff
@@ -1,22 +1,68 @@
...
 
 ## Installation
 
-Please refer to [Installation with Ascend NPU](../get_started/installation/install_ascend.md).
+VeOmni supports two installation methods for Ascend NPUs: `uv` (recommended for faster installation) and `pip`. Note that ARM architecture machines only support `uv` installation.
+
+### Installation Options
+
+- **x86 Architecture**: Supports both `uv` and `pip` installation methods
+- **ARM Architecture**: Only supports `uv` installation method
```
[the remark this answers is not in the document]
Comment (FoolPlayer) [reply docs/hardware_support/get_started_npu.md]: Agree

**Review by FoolPlayer — APPROVED** [review]
